api/integreatly/v1alpha1/grafanadashboard_types.go

```diff
@@ -33,10 +37,12 @@ import (
 // GrafanaDashboardSpec defines the desired state of GrafanaDashboard
 type GrafanaDashboardSpec struct {
 	Json             string                            `json:"json,omitempty"`
+	GzipJson         []byte                            `json:"gzipJson,omitempty"`
```

Comment:
We haven't been the best at doing this as you can see but let's start working on it. Please describe what GzipJson does.

Just put a comment or two above the definition that you create. For example:

// GzipJson is grafana dashboard json that is gziped and base64 encoded.

Reply:
Done. Also added similar ones to the original fields that they were based on to show the distinction.